x-pack/plugins/security_solution/public/detections/components/callouts/ml_job_upgrade_callout/index.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { memo } from 'react';
+import { CallOutMessage, CallOutSwitcher } from '../../../../common/components/callouts';
+import { useInstalledSecurityJobs } from '../../../../common/components/ml/hooks/use_installed_security_jobs';
+
+import * as i18n from './translations';
+
+const mlJobUpgradeCalloutMessage: CallOutMessage = {
+  type: 'primary',
+  id: 'ml-job-upgrade',
+  title: i18n.ML_JOB_UPGRADE_CALLOUT_TITLE,
+  description: <i18n.MlJobUpgradeCalloutBody />,
+};
+
+const MlJobUpgradeCalloutComponent = () => {
+  const { loading, jobs } = useInstalledSecurityJobs();
```

Comment:
As currently implemented, use of this hook will trigger additional ML calls on the page. We need to move this logic into a shared context a la `useUserData` in order to remove the duplicated/redundant calls to the ML API.